services/databases/postgresql/cnpg-cluster.yaml

```diff
@@ -6,13 +6,6 @@ metadata:
 spec:
   instances: 3
 
-  bootstrap:
-    initdb:
-      database: tts-db
-      owner: tts
-      secret:
-        name: tts-secret
-
```

Comment:
Why did you remove the bootstrap definitions?

Reply:
i needed a db for staging and i couldn't add a new bootstrap definition, so I put it in the managed section. I'm not sure I did the right thing and I didn't check if it was working. Will try to do it tomorrow.

Reply:
Okay, without the bootstrap definitions, it just creates a default database called "app", and there can only be one initdb in the bootstrap definitions.

Reply:
Having tts or not having bootstrap definitions will have the same effect, because the DB will need to be created in postgresql